maintenance/troubleshoot/vpp/tracing-packets.md

```diff
@@ -0,0 +1,272 @@
+---
+title: Tracing packets in VPP
+description: Specific instructions for packet tracing with the VPP dataplane
+canonical_url: '/maintenance/troubleshoot/vpp/tracing-packets'
+---
+
+Because VPP is handling all your traffic and not Linux, the usual troubleshooting tools (`tcpdump`, etc.) will not work as expected with the VPP dataplane. Fortunately however, VPP comes with its own built-in troubleshooting tools. This page describes how to trace packets in VPP, and can be useful to understand what is happening if your traffic is dropped somewhere in the cluster network.
```

Comment:
Because VPP is handling all your traffic and not Linux, the usual troubleshooting tools (`tcpdump`, etc.) will not work as expected with the VPP dataplane. Fortunately, VPP has its own built-in troubleshooting tools. This article describes how to trace packets in VPP, and understand what is happening if traffic is dropped somewhere in the cluster network.